src/card.tsx

```diff
@@ -393,3 +403,296 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
+            dataAttributes,
+            width,
+            height,
+            aspectRatio,
+            'aria-label': ariaLabel,
+        },
+        ref
+    ) => {
+        const withGradient = !!backgroundImage;
+        const textShadow = withGradient ? '0 0 16px rgba(0,0,0,0.4)' : undefined;
+        const hasTopActions = actions?.length || onClose;
+        return (
+            <MaybeWithActions
+                width={width}
+                height={height}
+                aspectRatio={aspectRatio}
+                onClose={onClose}
+                actions={actions}
+                aria-label={ariaLabel}
+                isInverse={isInverse}
+            >
+                <InternalBoxed
+                    borderRadius={16}
+                    className={styles.boxed}
+                    dataAttributes={dataAttributes}
+                    ref={ref}
+                    width="100%"
+                    minHeight="100%"
+                    isInverse={isInverse}
+                >
+                    <div
+                        className={styles.displayCard}
+                        style={{
+                            backgroundImage: backgroundImage
+                                ? `url("${CSS.escape(backgroundImage)}")`
+                                : undefined,
+                            paddingTop: withGradient && !icon && !hasTopActions ? 0 : 24,
+                        }}
+                    >
+                        {icon ? (
+                            <Box paddingBottom={withGradient ? 0 : 40} paddingX={24}>
+                                {icon}
+                            </Box>
+                        ) : (
+                            <Box paddingBottom={actions?.length || onClose ? (withGradient ? 24 : 64) : 0} />
+                        )}
+                        <Box
+                            paddingX={24}
+                            paddingTop={withGradient ? 40 : 0}
+                            paddingBottom={24}
+                            className={withGradient ? styles.displayCardGradient : undefined}
+                        >
+                            <Stack space={24}>
+                                <div>
+                                    <Stack space={8}>
+                                        {(headline || pretitle || title) && (
+                                            <header>
+                                                <Stack space={16}>
+                                                    {headline}
+                                                    <Stack space={4}>
+                                                        {pretitle && (
+                                                            <Text2
+                                                                forceMobileSizes
+                                                                truncate={pretitleLinesMax}
+                                                                as="div"
```

Comment:
if this is a pretitle, should this be read before the title? h2? or after the title h4?

Reply:
I wouldn't give it an h category